ci: add pr title check to enforce a consistent git history (#290)
tuono-labs/tuono: apps/documentation/src/routes/documentation/contributing/pull-requests.mdx

import MetaTags from '@/components/meta-tags'

<MetaTags

>   title="Tuono - Contributing"

Reviewer: ```suggestion
  title="Tuono - Opening a pull request"
```

  canonical="https://tuono.dev/documentation/contributing/pull-requests"
  description="Contribute to Tuono. Learn here how to open a pull request."
/>

import Breadcrumbs, { Element } from '@/components/breadcrumbs'

<Breadcrumbs
  breadcrumbs={[
    { label: '✨ Contributing', href: '/documentation/contributing' },
    { label: 'Pull Requests' },
  ]}
/>

# Pull Requests

Once your changes are ready, you can create a PR!

If you are not familiar with GitHub pull requests,
you can check the [official documentation](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/creating-a-pull-request).

---

> - If you are fixing a issue remember to link the issue itself in the PR description

Reviewer: ```suggestion
- If you are fixing an issue, bear in mind to link the issue itself in the PR description.
```

> - If a new feature is being added, describe the intended use case that feature fulfills.

Reviewer: ```suggestion
- If you are adding a new feature, describe the intended use case that the feature fulfills.
```

> Bug fixes and new features should include tests.

## Title

> The PR title must match the following format:

Reviewer: Shall we link here the conventional commit standard? https://www.conventionalcommits.org/en/v1.0.0/

Author: No, because we are not following the conventional commits standard. 
E.g.: We don't use `!` to mark breaking changes
